tests/avalon_mm/AvalonBFM.bsv

```diff
@@ -54,11 +54,11 @@ interface AvalonMM;
     interface AvalonmmSlave_0     slave_0;
 endinterface
 import "BVI" avlm_avls_1x1 =
-module mkAvalonMM#(Clock clk_clk, Reset reset_reset_n)(AvalonMM);
+module mkAvalonMM(AvalonMM);
     default_clock clk();
     default_reset rst();
-        input_clock clk_clk(clk_clk) = clk_clk;
-        input_reset reset_reset_n(reset_reset_n) = reset_reset_n;
+        input_clock (clk_clk) <- exposeCurrentClock;
```

Comment:
I see. I hadn't noticed that mkAvalonMM was an import "BVI".

You should be able to combine the default_clock and input_clock:
    default_clock clk(clk_clk);

This is shorthand for:

   default_clock clk(clk_clk) <- exposeCurrentClock;

According to the bsv-reference-guide page 138.

I guess we don't really have any examples of that.